migrations/20170620163533_create_questionnaires_table.js

```diff
@@ -0,0 +1,22 @@
+
+exports.up = function(knex) {
```

Comment:
The file and skeleton of migration is generated with `yarn knex -- migrate:make name_of_migration`. The rest is handwritten.

There are [tools](https://github.com/jwilm/knex-migrate) to generate more of the migration based on naming conventions (a la Rails), but I didn't want to throw that in the mix right now.

I'll update README to include details about creating migrations